charts/victoria-metrics-cluster/values.yaml

```diff
@@ -89,6 +87,7 @@ vmselect:
     envflag.enable: true
     envflag.prefix: VM_
     loggerFormat: json
+    httpListenAddr: "0.0.0.0:8481"
```

Comment:
Not sure if I understand why this option needs to be added?

Reply:
removed it